terraform-modules/k8s/k8s-cluster.tf

```diff
@@ -38,7 +45,7 @@ resource "google_container_cluster" "cluster" {
   # See https://www.terraform.io/docs/providers/google/r/container_cluster.html#master_auth
   master_auth {
     client_certificate_config {
-      issue_client_certificate = true
+      issue_client_certificate = "true"
```

Comment:
this change is a bit weird :)

Reply:
Yeah...I find that sometimes tf seems to treat `true` as the value `1` and `"true"` as the JSON value `true`. I can't prove it but I prefer `"true"` now.